static/app/components/events/interfaces/crashContent/exception/stackTrace.tsx

```diff
@@ -30,6 +32,17 @@ type Props = {
   threadId?: number;
 };
 
+// Render native stacktrace when:
+// 1. All frames have a native platform override in `frame.platform`, or
+// 2. The event platform is native
+function shouldRenderNativeContent({data, platform}: Pick<Props, 'data' | 'platform'>) {
+  const framePlatforms = uniq(data?.frames?.map(frame => frame.platform) ?? []);
```

Comment:
Can you put this in a Set? We don't need lodash for uniqueness check.

Reply:
Can do! I see now that you're cutting down on lodash usage in another PR which seems good.